MyClass.java

```diff
@@ -22,5 +24,67 @@ public static int pot(){
 	    }
 	    return pot; //returns value of "pot"
 	  }
+  
+    public static boolean userInputLeadersInFinalCount(){ //Returns whether leaders will be included in final count; Made by Andrew
+	boolean leadersIncludedFinalCount;
+        Scanner myObj = new Scanner(System.in);
+        
+        System.out.println("Are leaders included in the final count? (yes or no)");
+        String leaderIncludedFinalCount = myObj.nextLine();
+        
+        if (leaderIncludedFinalCount.equals("yes")){
+//            System.out.println("Leaders will be included in final count");
```

Comment:
Not a requirement but a challenge:

These seem like a good message to print if you want a -v/--verbose mode to your program. Then the logging/output can be used to troubleshoot any problems.

Copy the following function snippet and paste at bottom of this class. Replace your commented out `println`s with this function use instead.
```
    public static void verboseLogging(String output){
        if (verboseMode == true){
            System.out.println(output);
        }
    }
```

Then, figure out how to read a `-v` [command-line argument](https://www.tutorialspoint.com/Java-command-line-arguments#:~:text=A%20command%2Dline%20argument%20is,array%20passed%20to%20main(%20)) when your program starts and use that to set a `verboseMode` boolean class variable when main is called.

> Hint: There's an `args[]` parameter variable in the `main(String args[])` function definition. See the link I posted above to use it.

Reply:
I can implement this after this request gets merged.